catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -390,6 +395,69 @@ public void testExecuteQueryCacheDisabled() throws Exception {
                 .size(), is(0));
     }
 
+    @Test
+    public void testMaxQueryPaging() throws Exception {
```

Comment:
❓ Shouldn't we have more tests here, i.e., one when we reach the total number of hits, one when we reach the number requested and one we reach the hard-coded limit (`1,000`)?